incubator/crate/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+name: crate
+version: 0.1.0
+description: A Helm chart for CrateDB
```

Comment:
Could the description be made a little more verbose? Our docker images are tagged with the description "CrateDB is a distributed SQL database handles massive amounts of machine data in real-time.", perhaps something like this?

Reply:
got it!